src/applications/letters/tests/containers/LetterPageWrapper.unit.spec.jsx

```diff
@@ -15,4 +16,13 @@ describe('<LetterPageWrapper>', () => {
     const vdom = tree.getRenderOutput();
     expect(vdom).to.exist;
   });
+  it('should render an empty address warning on the view screen and disable the View letters button', () => {
+    const { getByText } = render(
+      <MemoryRouter initialEntries={[`/confirm-address`]}>
+        <LetterPageWrapper />
+      </MemoryRouter>,
+    );
+
+    expect(getByText('Your VA letters and documents').exist);
```

Comment:
Could we add an assertion here that looks for either the missing address element (or text) or confirms the View letters button is disabled?